capmvm/base-ubuntu/Makefile

```diff
@@ -0,0 +1,19 @@
+DOCKER := docker
+
+REGISTRY?=gcr.io/weaveworks
+IMAGE_NAME?=$(REGISTRY)/capmvm-ubuntu-base
```

Comment:
I think this image is really a flintlock base ubuntu image as there isn't anything specific to capmmvm yet. I think maybe we could change the parent folder from `capmvm` to `flintlock`. wdyt?